Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -180,31 +182,56 @@ def get_reply_body(notes, incident_id, attachments):
         return_error(get_error(res))
 
 
-def get_email_recipients(email_to, email_from, service_mail):
+def get_email_recipients(email_to, email_from, service_mail, mailbox):
     """Get the email recipient.
     Args:
         email_to (str): The email receiver.
         email_from (str): The email's sender.
         service_mail (str): The mail listener.
+        mailbox (str): The mailbox configured in the relevant integration.
     Returns:
         The email recipients.
     """
     email_to_set = {email_from}
     email_to = argToList(email_to)
     email_to_set = email_to_set.union(set(email_to))
-    service_mail_recipient = next(recipient for recipient in email_to_set if service_mail in recipient)
-    if service_mail_recipient:
-        email_to_set.remove(service_mail_recipient)
+
+    recipient_to_remove = ''
+    address_to_remove = mailbox if mailbox else service_mail
+    for recipient in email_to_set:
+        if address_to_remove and address_to_remove in recipient:
+            recipient_to_remove = recipient
+            break
```

Comment:
```suggestion
    if address_to_remove:
          for recipient in email_to_set:
               if address_to_remove in recipient:
                  recipient_to_remove = recipient
                  break
```

If address_to_remove is None we dont need to perform this loop